evm/src/cpu/kernel/asm/memory/packing.asm

```diff
@@ -42,10 +42,10 @@ global mload_packing_u64_LE:
 // Post stack: offset'
 global mstore_unpacking:
     // stack: context, segment, offset, value, len, retdest
-    %stack(context, segment, offset, value, len, retdest) -> (context, segment, offset, value, len, len, offset, retdest)
+    %stack(context, segment, offset, value, len, retdest) -> (context, segment, offset, value, len, offset, len, retdest)
     // stack: context, segment, offset, value, len, len, offset, retdest
```

Comment:
Nit: This comment now has the wrong order too. Up to you whether you adjust the comment or just remove it entirely (the `%stack` macro is communicating the same information in any case).